tests/test_lab_evaluate.py

```diff
@@ -0,0 +1,466 @@
+                "evaluate",
+                "--benchmark",
+                "mt_bench",
+            ],
+        )
+        assert (
+            result.output
+            == "Benchmark mt_bench requires the following args to be set: ['model', 'judge-model']\n"
+        )
+        assert result.exit_code != 0
+
+
+def test_missing_benchmark():
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            lab.ilab,
+            [
+                "--config=DEFAULT",
+                "model",
+                "evaluate",
+            ],
+        )
+        assert "Missing option '--benchmark'" in result.output
+        assert result.exit_code != 0
+
+
+def test_invalid_model_mt_bench():
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            lab.ilab,
+            [
+                "--config=DEFAULT",
+                "model",
+                "evaluate",
+                "--benchmark",
+                "mt_bench",
+                "--model",
+                "invalid",
+                "--judge-model",
+                "models/instructlab/merlinite-7b-lab",
+            ],
+        )
+        assert "Failed to determine backend:" in result.output
+        assert result.exit_code != 0
+
+
+def test_invalid_model_mmlu():
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            lab.ilab,
+            [
+                "--config=DEFAULT",
+                "model",
+                "evaluate",
+                "--benchmark",
+                "mmlu",
+                "--model",
+                "invalid",
+            ],
+        )
+        # TODO: This error could use some work
+        assert "is not a valid model" in str(result.exception)
+        assert result.exit_code != 0
+
+
+@patch(
+    "instructlab.model.evaluate.launch_server",
+    return_value=(mock.MagicMock(), "http://127.0.0.1:8000/v1"),
+)
+def test_invalid_taxonomy_mt_bench_branch(
+    launch_server_mock,
+):
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            lab.ilab,
+            [
+                "--config=DEFAULT",
+                "model",
+                "evaluate",
+                "--benchmark",
+                "mt_bench_branch",
+                "--model",
+                "models/instructlab/granite-7b-lab",
+                "--judge-model",
+                "models/instructlab/merlinite-7b-lab",
+                "--base-model",
+                "models/instructlab/granite-7b-lab",
+                "--branch",
+                "rc",
+                "--base-branch",
+                "main",
+                "--taxonomy-path",
+                "invalid_taxonomy",
+            ],
+        )
+        launch_server_mock.assert_called_once()
+        # TODO: This error could use some work
+        assert "/invalid_taxonomy" in str(result.exception)
```

Comment:
`"The taxonomy data is not in-line with what is expected" in str(result.exception)` or `"invalid taxonomy data" in str(result.exception)`

Reply:
Not quite sure what's being stated here.  This is another case the error handling is returning an exception rather than nice output.  The TODO is noting improvement is needed